### ISSUE TITLE
Pull Requests for Your Fork

### ISSUE THREAD

**Issue by ShubhamVerma1811** [hxdyj/vscode-react-outline#1]
I noticed that you've forked [my project](https://github.com/ShubhamVerma1811/vscode-react-outline) and added some great new features! 
I would love to see those changes integrated into the main repository. 
If you could submit those changes as pull requests and contribute to it, then that would be fantastic.

**Comment by hxdyj** [comment]
> I noticed that you've forked [my project](https://github.com/ShubhamVerma1811/vscode-react-outline) and added some great new features! I would love to see those changes integrated into the main repository. If you could submit those changes as pull requests and contribute to it, then that would be fantastic.

Of course, [here is](https://github.com/ShubhamVerma1811/vscode-react-outline/pull/20) 🤝
